### DISCUSSION TITLE
Custom generator with pagination

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [getpelican/pelican#3117] [Q&A]
Hi,

I want know and understand (if possible) if i can generate a list of something that isn't articles/blog, for example a list of projects or a list or other thing that i want using pelican engine with pagination.

For example, i have a list of projects that i want paginate and show in my website like this: https://devpah.com/

If it is possible, there is some example/tutorial about?

Thanks.

**Comment by Tseing** [comment]
It will be a little troublesome. Pelican can also process HTML files. These files will be processed like MARKDOWN files and display as articles in blog. If you would like to generate articles from existed HTML file, it is necessary for you to manually add some `<meta>` information into HTML files. For example,

```html
<html>
    <head>
        <title>My super title</title>
        <meta name="tags" content="thats, awesome" />
        <meta name="date" content="2012-07-09 22:28" />
        <meta name="modified" content="2012-07-10 20:14" />
        <meta name="category" content="yeah" />
        <meta name="authors" content="Alexis Métaireau, Conan Doyle" />
        <meta name="summary" content="Short version for index and feeds" />
    </head>
    <body>
        This is the content of my super blog post.
    </body>
</html>
```

More information in [Pelican documentation](https://docs.getpelican.com/en/latest/content.html).
